matRad_fluenceOptimization.m

```diff
@@ -100,33 +99,56 @@
 ixTarget       = ixTarget(i);
 wOnes          = ones(dij.totalNumOfBixels,1);
 
-% modified settings for photon dao
-if pln.propOpt.runDAO && strcmp(pln.radiationMode,'photons')
-%    options.ipopt.max_iter = 50;
-%    options.ipopt.acceptable_obj_change_tol     = 7e-3; % (Acc6), Solved To Acceptable Level if (Acc1),...,(Acc6) fullfiled
+% determine wOnes
+
+ 
+%This VMAT stuff is not clear to me why do we need it here?
+%We would need the stf only because of this part of the code...
+if ~isfield(pln.propOpt,'runVMAT')
+    pln.propOpt.runVMAT = false;
+end
 
+if pln.propOpt.runVMAT
```

Comment:
This if branch is the only reason we require the stf in the fluenceOptimization call.
The only thing it does is to set a start weight of 0 to beamlets of certain sub-beams. I don't completely get it, because the comment says it should avoid optimization there, but I don't think that setting certain wOnes to zero stops them from being used/changed during the optimization.